cmake/macros.cmake

```diff
@@ -202,7 +202,8 @@ macro(PDAL_ADD_TEST _name)
         ${PDAL_ADD_TEST_INCLUDES}
         ${PROJECT_SOURCE_DIR}/test/unit
         ${PROJECT_BINARY_DIR}/test/unit
-        ${PROJECT_BINARY_DIR}/include)
+        ${PROJECT_BINARY_DIR}/include
+        ${NLOHMANN_INCLUDE_DIR})
```

Comment:
I think we only want to add `${NLOHMANN_INCLUDE_DIR}` to individual tests. Or at least that's what we did in the past.

Reply:
We could, but it's pretty much every reader test plus quite a few others. I had a lot of them written out, and thought it would be cleaner to do this. Could definitely go back to the other style.